beacon-chain/blockchain/service.go

```diff
@@ -209,6 +228,32 @@ func (c *ChainService) updateHead(processedBlock <-chan *types.Block) {
 	}
 }
 
+// DEPRECATED: Will be replaced by new block processing routine
+func (c *ChainService) blockProcessingOld(processedBlock chan<- *types.Block) {
```

Comment:
Can we just delete this one? it has the same signature and appears to have results